packages/widgets/src/dockpanel.ts

```diff
@@ -68,13 +68,16 @@ class DockPanel extends Widget {
     this._mode = options.mode || 'multiple-document';
     this._renderer = options.renderer || DockPanel.defaultRenderer;
     this._edges = options.edges || Private.DEFAULT_EDGES;
+    if (options.tabsMovable !== undefined) {
+      this._tabsMovable = options.tabsMovable;
```

Comment:
I think I would just do:
```typescript
this._tabsMovable = !!options.tabsMovable; 
```

Reply:
Wouldn't that mean that an `undefined` would override the default `true`?

Reply:
Correct. I am dumb. I am used to thinking of defaults as `false`.

Reply:
In TSC 3.7 it would be:  `this._tabsMovable = ??options.tabsMovable;` (from memory)